external-builds/pytorch/repo_management.py

```diff
@@ -185,6 +185,7 @@ def apply_all_patches(
 
 
 # repo_hashtag_to_patches_dir_name('2.7.0-rc9') -> '2.7.0'
+# Note: unused function, delete? or use in get_patches_dir_name?
```

Comment:
quick search suggests this might be a copy left over of `experimental/rockbuilder/lib_python/repo_management.py`

maybe in the future one could add this unified to the `build_tools`?